Sources/ScryfallKit/Models/Card/Card.swift

```diff
@@ -33,7 +33,7 @@ public struct Card: Codable, Identifiable, Hashable, Sendable {
   /// The language that this specific printing was printed in
   public var lang: String
   /// A link to where you can begin paginating all re/prints for this card on Scryfall’s API.
-  public var printsSearchUri: String
+  public var printsSearchUri: String?
```

Comment:
[Scryfall's API docs](https://scryfall.com/docs/api/cards#core-card-fields) don't list this as an optional field, have you seen cards where this property is nil?